radio/src/pulses/modules_helpers.h

```diff
@@ -644,7 +644,13 @@ inline void resetMultiProtocolsOptions(uint8_t moduleIdx)
   else {
     g_model.moduleData[moduleIdx].multi.autoBindMode = 0;
   }
+
+#if defined (RADIO_Commando8)
+  g_model.moduleData[moduleIdx].multi.optionValue = -27;
```

Comment:
Because our D8 module crystal frequency is 26.000050MHZ, and the R81 receiver crystal frequency is 25.999340MHZ, so there is a frequency deviation and the frequency needs to be compensated. This variable g_model.moduleData[moduleIdx].multi.optionValue cannot save the setting value after switching to another module, so it is set to -27 by default. Is there a better approach for this?